docs/OTAnalytics/usage-ui.md

```diff
@@ -0,0 +1,434 @@
+
+    ![Show tracks intersecting the selected sections](usage-ui/show-tracks-intersecting-sections.png)
+
+=== "Not intersecting sections"
+    Tracks intersecting none of the selected sections are shown.
+
+    ![Show tracks not intersection the selected sections](usage-ui/show-tracks-not-intersecting-sections.png)
+
+=== "Assigned to flows"
+    Tracks assigned to at least one selected flow are shown.
+
+    ![Show tracks assigned to flows](usage-ui/show-tracks-assigned-to-flows.png)
+
+=== "Not assigned to flows"
+    Tracks not assigned to one of the selected flows are shown.
+
+    ![Show tracks not assigned to flows](usage-ui/show-tracks-not-assigned-to-flows.png)
+
+### Show start- and endpoints
+
+This group shows the start- and endpoints of tracks.
+The tracks to be shown can vary in the same flavors as described in [Show tracks](#show-tracks).
+
+=== "All"
+    Start- and endpoints of all tracks are shown.
+
+    ![Show start- and endpoints of all tracks](usage-ui/show-start-end-points-all.png)
+
+=== "Intersecting sections"
+    Start- and endpoints of tracks intersecting at least one of the selected sections are shown.
+
+    ![Show start- and endpoints of tracks intersecting the selected sections](usage-ui/show-start-end-points-intersecting-sections.png)
+
+=== "Not intersecting sections"
+    Start- and endpoints of tracks intersecting none of the selected sections are shown.
+
+    ![Show start- and endpoints of tracks not intersection the selected sections](usage-ui/show-start-end-points-not-intersecting-sections.png)
+
+=== "Assigned to flows"
+    Start- and endpoints of tracks assigned to at least one selected flow are shown.
+
+    ![Show start- and endpoints of tracks assigned to flows](usage-ui/show-start-end-points-assigned-to-flows.png)
+
+=== "Not assigned to flows"
+    Start- and endpoints of tracks not assigned to one of the selected flows are shown.
+
+    ![Show start- and endpoints of tracks not assigned to flows](usage-ui/show-start-end-points-not-assigned-to-flows.png)
+
+### Show detections of current frame
+
+This group visualizes the detections of a single frame in different flavors.
+
+=== "Bounding Box"
+    The bounding boxes of all detections in the currently visible frame are shown.
+
+    ![Show bounding box of detections in the current frame](usage-ui/show-detections-bounding-boxes.png)
+
+=== "Track point"
+    The track points of all detections in the currently visible frame are shown.
+
+    ![Show track points of detections in the current frame](usage-ui/show-detections-track-point.png)
+
+### Show events
+
+=== "Current filter"
+    The track points of all events within the filter range are shown.
+
+    ![Show events in the current filter](usage-ui/show-events-current-filter.png)
+
+=== "Current frame"
+    All events within the range of the current frame are shown.
+
+    ![Show events in the current filter](usage-ui/show-events-current-frame.png)
+
```

Comment:
The ‘Offset’ sliders on the left-hand side of the user interface (under the ‘Load tracks’ button) are for visualisation purposes only, right? They have no effect on any analyses? Then perhaps they should move to the right-hand side in the ‘Visualisation level’. In any case, their description is missing.